public/index.html

```diff
@@ -1,65 +1,42 @@
-        body { border-top: 16px solid #ffa100; }
-      }
-    </style>
+     <title>Log into ASB API</title>
+     <script src="https://www.gstatic.com/firebasejs/7.16.1/firebase-app.js"></script>
+     <script src="https://www.gstatic.com/firebasejs/7.16.1/firebase-auth.js"></script>
+     <script>
+        var firebaseConfig = {
+          
+        };
+        firebase.initializeApp(firebaseConfig);
+     </script>
+     <script src="https://www.gstatic.com/firebasejs/ui/4.6.0/firebase-ui-auth.js"></script>
+     <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/4.6.0/firebase-ui-auth.css" />
+     <script>
+        var ui = new firebaseui.auth.AuthUI(firebase.auth());
+        var uiConfig = {
+           signInSuccessUrl: 'user_data.html',
+           callbacks: {
+           signInSuccessWithAuthResult: function(authResult, redirectUrl) {
+              return true;
+           },
+           uiShown: function() {
+              document.getElementById('loader').style.display = 'none';
+           }
+         },
+        signInOptions: [
+           firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+           firebase.auth.EmailAuthProvider.PROVIDER_ID
+        ],
+        tosUrl: '<your-tos-url>',
+        privacyPolicyUrl: '<your-privacy-policy-url>'
+        };
+        ui.start('#firebaseui-auth-container', uiConfig);
+     </script>
   </head>
   <body>
-    <div id="message">
-      <h2>Welcome</h2>
-      <h1>Firebase Hosting Setup Complete</h1>
-      <p>You're seeing this because you've successfully setup Firebase Hosting. Now it's time to go build something extraordinary!</p>
-      <a target="_blank" href="https://firebase.google.com/docs/hosting/">Open Hosting Documentation</a>
-    </div>
-    <p id="load">Firebase SDK Loading&hellip;</p>
-
-    <script>
-      document.addEventListener('DOMContentLoaded', function() {
-        // // 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
-        // // The Firebase SDK is initialized and available here!
-        //
-        // firebase.auth().onAuthStateChanged(user => { });
-        // firebase.database().ref('/path/to/ref').on('value', snapshot => { });
-        // firebase.messaging().requestPermission().then(() => { });
-        // firebase.storage().ref('/path/to/ref').getDownloadURL().then(() => { });
-        //
-        // // 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
-
-        try {
-          let app = firebase.app();
-          let features = ['auth', 'database', 'messaging', 'storage'].filter(feature => typeof app[feature] === 'function');
-          document.getElementById('load').innerHTML = `Firebase SDK loaded with ${features.join(', ')}`;
-        } catch (e) {
-          console.error(e);
-          document.getElementById('load').innerHTML = 'Error loading the Firebase SDK, check the console.';
-        }
-      });
-    </script>
+     <h1>Success!  The example.com server block is working!</h1>
```

Comment:
Let's change it to "Welcome to Android Security Bulletin REST API. Please sign up.